test/src/basic.test.ts

```diff
@@ -14,32 +14,27 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-import { H160, H256, Parcel, U256, privateKeyToAddress } from "codechain-sdk";
+import { SDK } from "codechain-sdk";
 import { wait } from "./helper/promise";
 import CodeChain from "./helper/spawn";
 
-const secret = new H256("ede1d4ccb4ec9a8bbbae9a13db3f4a7b56ea04189be86ac3a6a439d9a0a1addd");
-const address = new H160(privateKeyToAddress(secret.value));
+const TIMEOUT = 10000;
 
-const instance = new CodeChain();
+const secret = "ede1d4ccb4ec9a8bbbae9a13db3f4a7b56ea04189be86ac3a6a439d9a0a1addd";
+const address = SDK.util.getAccountIdFromPrivate(secret);
 
-function payment(nonce: U256): Parcel {
-  const networkId = 17;
-  return Parcel.payment(
-    nonce,
-    new U256(10),
-    networkId,
-    new H160("3f4aa1fedf1f54eeb03b759deadb36676b184911"),
-    new U256("0")
-  );
-}
+const instance = new CodeChain();
 
 test("basic scenario", async () => {
   await instance.start("cargo", ["run", "--", "-c", "solo"]);
-  const nonce = await instance.sdk.getNonce(address);
-  const hash = await instance.sdk.sendSignedParcel(payment(nonce!).sign(secret));
+  const nonce = await instance.sdk.rpc.chain.getNonce(address);
+  const parcel = await instance.sdk.core.createPaymentParcel({
+    recipient: "3f4aa1fedf1f54eeb03b759deadb36676b184911",
+    amount: 0,
+  });
+  const hash = await instance.sdk.rpc.chain.sendSignedParcel(parcel.sign({ secret, nonce, fee: 10 }));
   await wait(3000);
```

Comment:
I think we can remove this line now.
Let's patch this in this PR.

Reply:
I removed that line.